app/models/game.rb

```diff
@@ -40,6 +40,14 @@ def checkmate_coords(x, y)
     end
   end
 
+  def white
```

Comment:
It was my understand that the way we have our model set up, these are necessary to populate one "player" on the board and all the color pieces they come with...